### ISSUE TITLE
Output html should include doctype if it is specified in the input html

### ISSUE THREAD

**Issue by dara76** [jackfranklin/html-dist#20]
html-dist is stripping the `doctype` declaration from the input html which can cause problems.

For example:

```
<!doctype html>
<html>
  <body>
    <svg height="100%" width="100%" viewBox="0 0 400 50" style="background-color:red;border:solid 1px black"></svg>
  </body>
</html>
```

and

```
<html>
  <body>
    <svg height="100%" width="100%" viewBox="0 0 400 50" style="background-color:red;border:solid 1px black"></svg>
  </body>
</html>
```

produce different results.

**Comment by jackfranklin** [comment]
👍 definitely a big bug. I'd welcome a PR or I'll try to get to having a look at this soon.

**Comment by jackfranklin** [comment]
Published in 0.3.1.
